.env.development

```diff
@@ -22,20 +23,21 @@ FAVICON_URL='https://edx-cdn.org/v3/default/favicon.ico'
 ALGOLIA_APP_ID=''
 ALGOLIA_SEARCH_API_KEY=''
 ALGOLIA_INDEX_NAME=''
-PORT=1991
-FEATURE_CODE_MANAGEMENT=true
-FEATURE_REPORTING_CONFIGURATIONS=true
-FEATURE_ANALYTICS=true
-FEATURE_SUPPORT=true
-FEATURE_SAML_CONFIGURATION=true
-FEATURE_EXTERNAL_LMS_CONFIGURATION=true
-FEATURE_BULK_ENROLLMENT=true
-FEATURE_SETTINGS_PAGE=true
-FEATURE_SETTINGS_UNIVERSAL_LINK=true
+PORT='1991'
+FEATURE_CODE_MANAGEMENT='true'
```

Comment:
Changed the values here to explicitly be strings, as that is how they're parsed (not an actual boolean).